pages/my-reports/script.js

```diff
@@ -0,0 +1,95 @@
+import MyReport from '../../assets/helpers/report-card.js'
+import { API_URL } from '../../constants.js'
+import { getUserSession } from '../../assets/helpers/storage.js'
+
+const user = getUserSession()
+let userID = user?.id
+let recentReportArr = []
+let olderReportArr = []
+const recentReports = document.getElementById('recentReports')
+const olderReports = document.getElementById('olderReports')
+const recentBtn = document.getElementById('recentReportsBtn')
+const olderBtn = document.getElementById('olderReportsBtn')
+
+// Checkbox toggle
+// Set the recentBtn to be checked initially
+recentBtn.checked = true
+displayRecentReports()
+displayOlderReports()
+
+recentBtn.addEventListener('click', () => {
+	recentReports.style.display = 'block'
+	olderReports.style.display = 'none'
+})
+olderBtn.addEventListener('click', () => {
+	recentReports.style.display = 'none'
+	olderReports.style.display = 'block'
+})
+
+// Get all the recent reports for the logged in user and display them
+
+async function getRecentReports() {
+	try {
+		recentReportArr.splice(0, recentReportArr.length)
+
+		const response = await fetch(
+			`${API_URL}/hazard-report?cursor=20&size=40&user_id=${userID}&type=recent`
```

Comment:
add a TODO here to handle pagination

Reply:
added TODO here https://github.com/whamester/WMDD-4885-Project/pull/24/commits/b01cb10578117f476db5bcc7331b50f3785ab282